### ISSUE TITLE
Heroes V

### ISSUE THREAD

**Issue by dredknight** [sfidanza/arthur#2]
Hello Sfidanza,

I wonder if you are the same person as the user on [heroescommunity](http://heroescommunity.com/member.php3?action=viewprofile&UserName=sfidanza) ? I bumped into you by doing some research on your work on NCF framework.

How did you manage to edit the index.bin file that enables the players to see new objects in the map editor?
We want to add new objects inside (artifacts and may be more) but unfortunately no trace of the technical solution can be found on the forums.

If you are not the same person, please forgive me for bothering.

**Comment by sfidanza** [comment]
Hello,
I am indeed the same person. It was however quite some time ago. ;-)
I was in contact at the time with another community member who came up with the .exe hex patch on both the game and the map editor if I remember correctly. I do not seem to find our conversations anymore (maybe it was on some forum - I found no traces in my email from back then). So I am afraid I cannot help you much.

The only thing I can suggest is to try the patched .exe for the map editor that we provided in the NCF package (which I assume you tried already), as this allowed us to see additional creatures. If I remember correctly, we only updated the max number of creatures in there, so unless there is the same kind of limit for other objects, simply following the other guidelines should work (putting .pak files in the data/ folder).

I hope this helps a bit and sorry to not be of more help.
